dataland-frontend/src/components/general/apiKey/CreateApiKeyCard.vue

```diff
@@ -51,8 +50,8 @@
         >
```

Comment:
We could avoid people from picking dates in the past by modifying the props of the Calendar-element:

```Html
<Calendar v-model="value" :minDate="minDateValue" maxDate="maxDateValue" />
```

minDate should be the date of today

Reply:
This would also ease some other things (see my other comments) since you have one validty check less to do.

Reply:
done